types/src/v0/impls/stake_table.rs

```diff
@@ -1557,6 +1559,100 @@ impl EpochCommittees {
         Ok(block_reward_u256.into())
     }
 
+    pub async fn get_block_reward(
+        epoch: Option<Epoch>,
```

Comment:
Don't quite understand why this is optional. In what case do we call this function without epoch?